BOJ/240114_알파벳/kwaksj329.js

```diff
@@ -0,0 +1,36 @@
+const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
+const [R, C] = input[0].split(' ').map(Number);
+const board = input.slice(1);
+
+function solution(R, C, board) {
+  let maxAnswer = 1;
+  const visited = Array(26).fill(false);
+  visited[board[0][0].codePointAt() - 65] = true;
+
+  function findNext(position, visited, answer) {
+    let [y, x] = position;
+    if (answer > maxAnswer) maxAnswer = answer;
```

Comment:
```suggestion
    maxAnswer = Math.max(maxAnswer, answer);
```
이렇게 하면 좀 더 간단할 것 같아요!